content/rest/overview/authenticating-to-the-rest-api.md

```diff
@@ -67,7 +67,7 @@ curl --request POST \
 --url "{% data variables.product.api_url_code %}/applications/YOUR_CLIENT_ID/token" \
 --user "YOUR_CLIENT_ID:YOUR_CLIENT_SECRET"{% ifversion api-date-versioning %} \
 --header "Accept: application/vnd.github+json" \
---header "X-GitHub-Api-Version: {{ allVersions[currentVersion].latestApiVersion }}"{% endif %}
+--header "X-GitHub-Api-Version: {{ allVersions[currentVersion].latestApiVersion }}"{% endif %} \
```

Comment:
Since this line is only included in some versions of the docs, let's put the `\` in the if/endif logic
```suggestion
--header "X-GitHub-Api-Version: {{ allVersions[currentVersion].latestApiVersion }}"\{% endif %} 
```